src/main/java/frc/robot/subsystems/Spitter2022.java

```diff
@@ -0,0 +1,109 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotConstants;
+import frc.robot.motors.MotorControllerEncoder;
+import frc.robot.motors.MotorControllerFactory;
+import frc.robot.motors.MotorType;
+
+/** The spitter subsystem, contains the flywheel and its motor only. */
+public class Spitter2022 extends SubsystemBase {
+
+  private final double THRESHOLD = 2;
```

Comment:
This name doesn't really tell me what it does. Maybe a better name would be `SHOOTING_SPEED_TOLERANCE`.
Also, `... = 2.0;`